code/mob/living/carbon/human/normal.dm

```diff
@@ -96,6 +96,11 @@
 		..()
 		JobEquipSpawned("Chaplain")
 
+/mob/living/carbon/human/normal/artist
```

Comment:
?

Reply:
Right. This is technically a leftover from a corpse job preset, and since those use the defines in `normal.dm` and artist was missing, I added it, but we can just remove it entirely.